src/modules/account/account.service.ts

```diff
@@ -119,12 +119,24 @@ export class AccountService {
   public async delete(id: number): Promise<void> {
     const account = await this.accountRepository.findOne({
       where: { id },
+      relations: {
+        instructionElements: true,
+      },
     })
 
     if (!account) {
       throw new NotFoundException(businessException([ERRORS.account.notFound]))
     }
 
+    if (account.instructionElements.length > 0) {
+      const instructionElementErrors = account.instructionElements.map((element) => ({
```

Comment:
It's better to create one error instead of array in this case and clarify all used instructions in the message